types/api/APIRequests.go

```diff
@@ -48,6 +48,11 @@ type APIRequestURL struct {
 	Wait bool         `json:"wait" form:"wait" query:"wait"`
 }
 
+type APIRequestKeyPageIndices struct {
+	APIRequestURL
+	Key string `json:"key" form:"key" query:"key" validate:"required"`
+}
+
```

Comment:
This should be removed. As far as I can tell, it isn't used anywhere, and regardless, types for API v2 should be added to `internal/api/v2/types.yml`.